lbs/storage.c

```diff
@@ -100,7 +101,7 @@ storage_init(const char * storagedir, size_t blocklen, long latency,
 		}
 
 		/* Does it have an integer number of blocks? */
-		if (sf->len % S->blocklen) {
+		if ((remainder = ((size_t)sf->len % S->blocklen))) {
```

Comment:
On second thought, it's probably worth adding an explicit `!= 0` here.

I'm also suspicious about the comment.  Isn't this checking for a **non**-integer number of blocks?  I mean, `a % b` will be non-zero if there's a remainder.